src/styles/GlobalStyle.js

```diff
@@ -0,0 +1,25 @@
+import { createGlobalStyle } from "styled-components";
+import "normalize.css";
+
+const GlobalStyle = createGlobalStyle`
+  @import url('https://fonts.googleapis.com/css?family=Lato:300,400,700&subset=latin-ext');
+
+  body {
+    font-family: "Lato", sans-serif;
+    font-size: ${props => props.theme.font.size.base};
+  }
+
+  img {
+    max-width: 100%;
+    height: auto;
+    display: block;
+  }
+
+  p {
+    color: ${props => props.theme.color.text.base};
+    line-height: ${props => props.theme.font.lineHeight.paragraph};
+    margin: 1em 0 1.5em;
```

Comment:
How about we stay consistant with either em or px

Reply:
This case makes sense as margins before and after text are most often based on the size of given text. So I'd leave that as is, as a general rule.

Reply:
aight!